Src/BendAssist.App/BendAssists/CornerRelief.cs

```diff
@@ -12,136 +12,56 @@ public sealed class CornerRelief : BendAssist {
    #region Methods --------------------------------------------------
    public override void Execute () {
       if (mPart is null) return;
-      List<Point2> intersectPts = [], // Intersect points between the two bend lines.
-                   pLinesStartPts = [], // All starting points of the PLines.
-                   pLinesEndPts = []; // All ending points of the PLines.
-      if (mPart.BendLines.Count >= 2) {
-         for (int i = 0; i < mPart.BendLines.Count - 1; i++) {
-            var (p1, cP1) = GetIntersectBendPts (mPart.BendLines[i].StartPoint, i);
-            if (p1 > 0) intersectPts.Add (cP1);
-            var (p2, cP2) = GetIntersectBendPts (mPart.BendLines[i].EndPoint, i);
-            if (p2 > 0) intersectPts.Add (cP2);
+      List<PLine> initialPlines = mPart.PLines; // An initial stage of the plines of the part is present in it.
+      Point2 p1 = new (), p2 = new (), p3 = new ();
+      var halfBA = mBendAllowance * 0.5;
+      // To return the new processed part when the part has one or more corner relief options.
+      bool assisted = false;
+      foreach (var info in mPart.AssistInfo) {
+         if (info.ReqAssist != EBendAssist.CornerRelief) continue;
+         assisted = true;
+         var vertex = mPart.Vertices.Find (v => v.Index == info.VIndex);
+         var quadrant = vertex.Quadrant (mPart.Centroid);
+         var (sDx, sDy, eDx, eDy) = (0.0, 0.0, 0.0, 0.0);
+         // To find the corner relief vertices by translate vertex using following vectors.
+         var (v1, v2, v3) = (new Vector2 (), new Vector2 (), new Vector2 ());
+         switch (quadrant) {
+            case IQuadrant.I or IQuadrant.III:
+               (v1, v2, v3) = (new (halfBA, 0), new (-halfBA, -halfBA), new (0, halfBA));
+               sDy = eDx = quadrant is IQuadrant.I ? halfBA : -halfBA;
+               break;
+            case IQuadrant.II or IQuadrant.IV:
+               (v1, v2, v3) = (new (0, halfBA), new (halfBA, -halfBA), new (-halfBA, 0));
+               (sDx, eDy) = quadrant is IQuadrant.II ? (-halfBA, halfBA) : (halfBA, -halfBA);
+               break;
+            default: break;
          }
+         if (quadrant is IQuadrant.III or IQuadrant.IV) (v1, v2, v3) = (v1 * -1.0, v2 * -1.0, v3 * -1.0);

```

Comment:
This could be as follows:

 if (quadrant is IQuadrant.III or IQuadrant.IV) (v1, v2, v3) = (v1 * -1.0, v2 * -1.0, v3 * -1.0);
 (p1, p2, p3) = (vertex + v1, vertex + v2, vertex + v3);
 int index1 = info.PLIndices.First (), index2 = info.PLIndices.Last ();
 var plines = mPart.PLines.Where (p => p.Index == index1 || p.Index == index2);
 var pl1 = (PLine)plines.First ().Trimmed (endDx: eDx, endDy: eDy);
 var pl2 = (PLine)plines.Last ().Trimmed (startDx: sDx, startDy: sDy);
 var pl3 = new PLine (p1, p2, index1);
 var pl4 = new PLine (p2, p3, index2);
 initialPlines = initialPlines.Where (p => p.Index != index1 && p.Index != index2).ToList();
 initialPlines.InsertRange (index1-1, [pl1, pl2, pl3, pl4]);